### DISCUSSION TITLE
Generator filter: exclude if present on named playlist

### DISCUSSION THREAD

**Discussion by da2x** [Taiko2k/Tauon#727] [Q&A]
Is there a way to exclude songs from one playlist songs that are present on another playlist? Something like `sx"Christmas"` (source exclude songs on the named playlist Christmas). I don’t see anything obvious on the [generators wiki](https://github.com/Taiko2k/TauonMusicBox/wiki/Generator-Codes).

The use case is to generate a top songs list from all sources that excludes the Christmas playlist pests.

**Comment by Taiko2k** [comment]
There wasn't, but I've now committed this feature. The new code is `px`, so, `px"christmas"`. (Its a filter, not a 'selector', so goes after the 'a')
